.github/workflows/tasklist-merge-ci.yml

```diff
@@ -10,14 +10,6 @@ concurrency:
   group: "${{ github.workflow }}-${{ github.ref }}"
 
 jobs:
-  run-build:
-    name: run-build
-    uses: ./.github/workflows/tasklist-ci-build-reusable.yml
```

Comment:
this also builds tasklist docker image, which is not the case in the unified ci

Reply:
For me it is still unclear whether we want or need this as this can also be done via camunda and the right profile ahould be enough for development

Reply:
Do you think we or the team can leverage the Camunda docker image instead of the Tasklist image?

On release, we would still create the tasklist image btw. But I think this would reduce a lot of complexity of our CIs if we reduce the docker image builds.

Wdyt @cmur2 ? 🤔 


But I guess might be an issue for SaaS and snapshot usage their 🤔 we might need to adjust then also something 🤔 Hm no sure anymore :D

Reply:
The `operate-merge-ci` also builds the Operate Docker image, so same question for that. If we want to keep mental load low, we can migrate them as well to the unified CI in their current form.

Reply:
It shouldn't be much effort to migrate those Docker build jobs in their current form (no snapshot build/push needed), can also help with that.

Reply:
Ok lets do that then.

Reply:
Actually, the `{operate|tasklist}-ci-buid-reusable` flows do not just build the Docker image, they also push that and the snapshot Maven artifacts to the registries so replacing is a bit more effort.To wrap this PR up I think it makes sense to not delete this nor the operate-merge-ci.yml yet and keep migrating in another PR - in the meantime we have some duplication but that is acceptable temporarily.

Reply:
Ok I will bring it back then.